### DISCUSSION TITLE
Active NOTAMS and Airspaces

### DISCUSSION THREAD

**Discussion by benniAvia** [Akaflieg-Freiburg/enroute#429] [Ideas]
Dear all, absolutely fantastic app. 

BTW, not a regular user of github, so created a poll by mistake! Apologies!

May I ask **_three_** questions: 

1. I see all the restricted, danger and other areas, and that clutters up quite a bit on the screen, is there a way to see only those that are active? E.g. those that are not active could be grey?

2. How do I see the NOTAMS?

3. I have Enroute on my phone and the iPad, is there a way to share the Routes and Waypoints between them in an easier manner?

Best regards,
Ben

**Comment by kebekus** [comment]
Dear benniaAvia,

your questions:

1. The open databases that we use do not contain information about which airspace is open and which isn't. 
2. Enroute is no substituts for proper flight preparation, so you need to check NOTAMs with the relevant official source for your country. For convenience, Enroute will show NOTAM locations in the map ([see the manual](https://akaflieg-freiburg.github.io/enrouteManual/05-referencePages/01-mainPage.html#waypoints-and-notams)) but it would be extremely unwise to rely on this information.
3. Yes, there is. You can share data with other apps, including file transfer apps. I do not know how that works exactly, because I do not own any Apple devices. @schneisi Can you help our user?

Best,  

Stefan.

**Comment by benniAvia** [comment]
Dear Stefan,

Many thanks, so I was flying today. 

I wonder: Is there a difference between the Android and iOS versions? At least on my ios devices (both iPhone and iPad), I could not see NOTAMS at all, but I know they are there (gliding), and another one (airport closed), but those were not visible. 

I have read the manual, but I am sure I am perhaps still making some mistake?

**Comment by kebekus** [comment]
There was a difference for a while. If you update to the latest version, the difference should be gone.

Best,

Stefan.

Am 19. August 2024 19:54:28 MESZ schrieb benniAvia ***@***.***>:
